src/spindump_main_lib.c

```diff
@@ -148,6 +148,7 @@ spindump_main_configuration_defaultvalues(struct spindump_main_configuration* co
   config->reportQlLoss = 0;
   config->reportPackets = 0;
   config->reportNotes = 1;
+  config->reportMinimumRtt = 1;
```

Comment:
should be initialized to 0 by default so that the value dos not get reported unless explicitly configured to do so.